java/bench/spark/pom.xml

```diff
@@ -141,7 +141,7 @@
     <dependency>
       <groupId>org.objenesis</groupId>
       <artifactId>objenesis</artifactId>
-      <version>2.6</version>
+      <version>3.3</version>
```

Comment:
It seems that we missed this at ORC-1207. Apache Spark 3.3.0 uses objenesis 3.2.

We need 3.2 instead of this PR. Could you make a PR, @williamhyun ?